uaclient/cli.py

```diff
@@ -320,6 +323,20 @@ def print_version(_args=None, _cfg=None):
     print(version.get_version())
 
 
+def action_refresh(args, cfg):
+    if not cfg.is_attached:
+        print(ua_status.MESSAGE_UNATTACHED)
+        return 1
+    contract_token = cfg.read_cache('contract-token')
+    if contract_token:
```

Comment:
If this is false, should we `return 1` at this point, rather than attempting a (presumably doomed?) `request_contract_updates` call?

Reply:
added a check and returned 1 if contract_token is falsish